### DISCUSSION TITLE
ChatGPT-writer email context (The previous email text) isn't auto-loaded

### DISCUSSION THREAD

**Discussion by alandardic** [GorvGoyl/ChatGPT-Writer#38] [Report Bugs and Issues]
Hi, 

I'm using your amazing tool, but the previous email text isn't auto-loaded for me.

Could you please help troubleshoot this issue?

Many thanks!
Alan

**Comment by alandardic** [comment]
Hi,

I just noticed it doesn't work only when opening the reply in a new window.

When replying on the original window, it indeed auto-loads the previous email

**Comment by GorvGoyl** [comment]
ah, thanks for reporting.. will fix it soon
